requirements.txt

```diff
@@ -1 +1,2 @@
 datajoint>=0.13
+pynwb
```

Comment:
Should we version lock so that we know which version our export function is tested to work with?  If we are not ready to lock the version, I suggest creating a GitHub Issue to remind us.

Reply:
Yes I think we should version lock this, to whatever is the latest version now, after tests passed.

Reply:
My local install of pynwb 1.4.0 has passed all tests. The latest version is 2.0.0., which I recall having issues with dependency matching. During alpha testing, I can revisit this issue and document the nature of the conflict. For now, I would pin to 1.4.0

Reply:
Thanks.  I have created Issue #18.